### DISCUSSION TITLE
courses/beginners-web-handbook/javascript-grammar/comparison-operators

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#55] [Q&A] [answered]
# courses/beginners-web-handbook/javascript-grammar/comparison-operators

바닐라코딩

https://www.vcdr.club/courses/beginners-web-handbook/javascript-grammar/comparison-operators

**Comment by jin-ttao** [comment, marked as answer]
안녕하세요. 혹시 누락이 있는지 싶어 여쭈어요! 
"2. 같다는 의미" 부분에서 예시가 따로 있을지, "3. 일치연산자" 부분에 설명이 따로 있을까요?

**Reply by jin-ttao** [reply]
아 일치 연산자, 동등 연산자가 '2. 같다는 의미'에 포함되는것으로 이해했습니다!

**Reply by Ken123777** [reply]
넵넵 맞습니다 🙇🏻‍♂️
